domains/aswincreates.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "AswinCreates",
+           "email": "aswinrout94281@gmail.com",
+           "discord": "952073733705724026"
+        },
+    
+        "record": {
+            "TXT": "dh=0613b355bada9aa6ed290515e3510ccfb7706602"
```

Comment:
This isn't supposed to be for the main domain, please use either A, CNAME or URL records to point to your website

Reply:
i followed all steps that is shown in docs :/

Reply:
> i followed all steps that is shown in docs :/

You made 2 files exactly like this. Main domain shouldn't have the discord txt verification, please point it to your website

Reply:
alright, what if i delete all files and recreate? will it work??

Reply:
> alright, what if i delete all files and recreate? will it work??

You know you can just replace the records of the file I requested changes on, right? You can also read the docs on what records we support

Reply:
alr